herbert_core/utilities/classes/@unique_references_container/unique_references_container.m

```diff
@@ -93,60 +93,60 @@
     % implement its storage. Ensure that any changes here in
     % unique_references_container will continue to be supported by unique_objects_container
     % if required.
-    
+
     properties (Access = protected)
         idx_ = zeros(1,0); %  array of unique global indices for each object stored
         stored_baseclass_ = ''; % the baseclass
         global_name_; % name of category referencing a global container backing this one
     end
-    
+
     properties (Dependent)
-        
+
         % saveable fields for save/loadobj
         stored_baseclass;
         global_name;    % category name for singleton storage
         unique_objects; % returns unique_objects_container
-        
+
         % other dependent properties
         n_unique_objects; % size of unique_objects (without creating it)
         idx; % object indices into the global unique objects container.
         n_objects; % numel(idx)
     end
-    properties(Dependent,Hidden)
-       n_runs;    % same as n_objects, provides a domain-specific interface
-                   % to the number of objects for SQW-Experiment
-                   % instruments and samples        
+    properties(Dependent,Hidden=true)
```

Comment:
These are the same. Why this change?

Reply:
for clarity and in attempt to resolve the identified issues.